docs/API/rating.md

```diff
@@ -44,14 +44,17 @@ const WATER_IMAGE = require('./water.png')
-| imageSize | 50 | number | The size of each rating image (optional) |
-| showRating | none | boolean | Displays the Built-in Rating UI to show the rating value in real-time (optional) |
-| style | none | function | Exposes style prop to add additonal styling to the container view (optional) |
+| prop                  | default         | type           | description                                                                                                                  |
+| ----                  | ----            | ----           | ----                                                                                                                         |
+| **onFinishRating**    | none            | function       | Callback method when the user finishes rating. Gives you the final rating value as a whole number **(required)**             |
+| type                  | star            | string         | Choose one of the built-in types: `star`, `rocket`, `bell`, `heart` or use type `custom` to render a custom image (optional) |
+| ratingImage           | star            | string         | Pass in a custom image source; use this along with `type='custom'` prop above (optional)                                     |
+| ratingColor           | #f1c40f         | string (color) | Pass in a custom fill-color for the rating icon; use this along with `type='custom'` prop above (optional)                   |
+| ratingBackgroundColor | white           | string (color) | Pass in a custom background-fill-color for the rating icon; use this along with `type='custom'` prop above (optional)        |
+| ratingCount           | 5               | number         | The number of rating images to display (optional)                                                                            |
+| imageSize             | 50              | number         | The size of each rating image (optional)                                                                                     |
+| showRating            | none            | boolean        | Displays the Built-in Rating UI to show the rating value in real-time (optional)                                             |
+| readonly              | false           | boolean        | Whether the rating can be modiefied by the user (optional)                                                                   |
+| startingValue         | ratingCount / 2 | number         | The initial rating to render (optional)                                                                                      |
+| fractions             | undefined       | number         | The number of decimal places for the rating value; must be between 0 and 20 (optional)                                       |
+| style                 | none            | function       | Exposes style prop to add additonal styling to the container view (optional)                                                 |
```

Comment:
looks good. 

if you have time, can you add a couple sample use cases to show using the `showRating` and `readonly` props.. as well as using `fractions`.

If not, don't worry.. I can add some myself once I merge this in.